mteb/tasks/BitextMining/multilingual/TedTalksBitextMining.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import annotations
+
+from typing import Any
+
+from datasets import DatasetDict, load_dataset
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskBitextMining, CrosslingualTask
+
+# Mapping from ISO 639-3 + script to ISO 639-1 used in TED
+_ISO6393_to_ISO6391 = {
+    "ara_Arab": "ar",
+    "aze_Latn": "az",
+    "bel_Cyrl": "be",
+    "bul_Cyrl": "bg",
+    # "ben_Beng": "bn",
```

Comment:
Why are these languages commented? Should they be removed?

Reply:
I think it is just for the speed of testing. I assume they will be commented out before the PR.

Reply:
I accidentally included TedTalks file (form #386) here, I removed it now.